bodhi/server/consumers/automatic_updates.py

```diff
@@ -139,10 +139,21 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
                 dbsession.add(user)
 
             log.debug(f"Creating new update for {bnvr}.")
+            changelog = generate_changelog(build)
+            if changelog:
+                notes = f"""Automatic update for {bnvr}.
+
+##### **Changelog**
+
+```
+{generate_changelog(build)}
```

Comment:
You could use `changelog` directly instead of calling `generate_changelog()` twice here.

Reply:
done!